cluster/saltbase/salt/kubelet/default

```diff
@@ -50,8 +50,9 @@
 {% set config = "--config=/etc/kubernetes/manifests" -%}
 
 {% set manifest_url = "" -%}
```

Comment:
You also need to default manifest_url_header and include it in the last line of this file (in `DAEMON_ARGS`)